topology/probes/ovsdb/ovsdb.go

```diff
@@ -325,16 +326,22 @@ func (o *Probe) OnOvsInterfaceAdd(monitor *ovsdb.OvsMonitor, uuid string, row *l
 	defer o.Ctx.Graph.Unlock()
 
 	intf := o.Ctx.Graph.LookupFirstNode(graph.Metadata{"UUID": uuid})
-	if mac != "" || attachedMAC != "" {
-		var macFilter *filters.Filter
-		if mac != "" {
-			macFilter = filters.NewTermStringFilter("MAC", mac)
-		} else {
-			macFilter = filters.NewTermStringFilter("MAC", attachedMAC)
+	if oldMAC != "" || mac != "" || attachedMAC != "" {
+		var macFilters []*filters.Filter
+		if oldMAC != "" {
```

Comment:
any reason to not use the mac or attachedMAC if the previous MAC was empty ?

Reply:
The behaviour I observed was:
- The interface is first seen by netlink but with a strange MAC address
- The OVS probe then handles the update. It has a valid MAC address but no previous MAC. So we don't have any other way to find the existing interface (neither with `mac`nor `oldMAC`).

Reply:
I don't see why we need to ensure that we have a previous MAC ? If the netlink one is incorrect and the OVS is correct then they are different and the `lookup` will fail.

Anyway let merge this version which fixes the CI and I'll try to submit another patch to understand this behavior.

Reply:
Agreed :-)